server/index.js

```diff
@@ -25,3 +26,34 @@ app.get('/api/rooms/:roomId', function (req, res) {
       res.json(data);
     });
 });
+
+app.post('/api/rooms/:roomId', (req, res) => {
+  const newReview = req.body;
+  console.log(newReview);
+  Review.create({
+    roomId: req.params.roomId,
```

Comment:
Hi Elana! API endpoint look great to me but for which database is it? Looks like mongo and according to Eric we should not be using the DB that we inherited